components/mixins/experience-event/experienceevent-card-action-details.schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-card-action-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Card Action Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card action details such as card type as well as activation and lock status.",
+  "definitions": {
+    "card-action-details": {
+      "properties": {
+        "xdm:cardType": {
+          "title": "Card Type",
+          "enum":["credit", "debit", "prepaid", "paypal", "atm", "virtual", "hsa", "fsa"],
+          "description": "Tracks the type of card associated with an event."
+        },
+        "xdm:cardActivated": {
+          "title": "Card Activated Success",
+          "type": "measure",

```

Comment:
Please refer the datatype using $ref. Same comment applies everywhere else wherever applicable.